### PR TITLE
feat: mount checks on a folder level

### PR DESCRIPTION
**Pull request by zackpollard**

### CODE REVIEW

File: server/src/services/storage.service.ts [hunk 1]
```diff
@@ -19,25 +20,36 @@ export class StorageService extends BaseService {
...
             this.logger.log(`Writing initial mount file for the ${folder} folder`);
             await this.createMountFile(folder);
           }
 
           await this.verifyReadAccess(folder);
           await this.verifyWriteAccess(folder);
+
+          if (!flags.mountChecks) {
+            flags.mountChecks = {};
+          }
```
Comment (danieldietzler) [inline server/src/services/storage.service.ts]: Why is this necessary? That should be defined either way in line 23-25, no?

**Review by danieldietzler — APPROVED** [review]

**Review by jrasm91 — APPROVED** [review]
